.github/workflows/updateprojects.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3                                                                                                  
+#                                                                                                                       
+# Copyright this project and its contributors                                                                          
+# SPDX-License-Identifier: Apache-2.0                                                                                   
+#                                                                                                                       
+# encoding=utf8
+
+import csv
+import urllib.request
+import json
+import os
+
+projectsCsvFile = os.path.dirname(os.path.realpath(__file__))+'/../../_data/projects.csv'
+
+landscapeBaseURL = 'https://landscape.aswf.io'
+landscapeHostedProjects = landscapeBaseURL+'/data/exports/projects-hosted.json'
+landscapeSingleItem = landscapeBaseURL+'/data/items/{}.json'
+
+# map landscape ID to slug
+keyMapping = {
+    'open-mainframe-education': 'mainframe-open-education' 
```

Comment:
Does this need to be updated for ASWF?

Reply:
Yep - actually removed entirely as it's not used. Done now